src/views/home/components/HistoryWatchVideoCard.tsx

```diff
@@ -45,33 +45,58 @@ const HistoryWatchVideoCard: FC<HistoryWatchVideoCardProps> = ({
   onlineCourseId,
   categoryName
 }) => {
+  const theme = useTheme()
+  const desktop = useMediaQuery(() => theme.breakpoints.up('xmd'))
+
+  const WatchAgainBtn = () => (
```

Comment:
vynn nanggung banget Btn
```suggestion
  const WatchAgainButton = () => (
```

Reply:
😁

Reply:
rojer sumatra